frappe/boot.py

```diff
@@ -258,8 +258,8 @@ def get_unseen_notes():
 			(select user from `tabNote Seen By` nsb
 				where nsb.parent=`tabNote`.name)''', (frappe.utils.now(), frappe.session.user), as_dict=True)
 
-def get_gsuite_status():
-	return (frappe.get_value('Gsuite Settings', None, 'enable') == '1')
+def get_google_drive_status():
+	return True if frappe.db.exists("Google Drive", {"enable": 1}) else False
```

Comment:
this looks redundant, why not just have:

```suggestion
	return frappe.db.exists("Google Drive", {"enable": 1})
```

Reply:
Not using boot anymore.